### ISSUE TITLE
[BUG] kirby-reorder-list | reorder-three-outline.svg not found

### ISSUE THREAD

**Issue by RasmusTraeholt** [kirbydesign/designsystem#2121]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
 
Upon first init of the kirby-reorder-list a not found error is thrown for a svg named reorder-three-outline

## Describe how to reproduce the bug
1. Go to [investment/overview](https://jyskebank-staging.relationsbank.bankdata.dk/drb/investment/overview)
2. Click on Rediger button
3. See error --> reorder-three-outline.svg not found, in browser console

## Which Kirby version was used?
5.0.18

## What was the expected behavior?

No error to be thrown

## Add any screenshots

![Screenshot 2022-03-23 at 10 57 36](https://user-images.githubusercontent.com/23026215/159674652-da9ee0a0-746b-4faa-9bd4-ef38c1fb2c33.png)

## Please complete the following information:
- OS: [e.g. Windows, MacOS, iOS]
- Browser [e.g. Chrome, Safari]
- Browser version [e.g. 22]

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/stable/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/stable/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/stable/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [stable branch](https://github.com/kirbydesign/designsystem/tree/stable) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Does this happen locally or in production as well? Otherwise, it is likely to be related to other icon issues we experience, and might be a duplicate of https://github.com/kirbydesign/designsystem/issues/1565.

**Comment by RasmusTraeholt** [comment]
The issues does not seem to be related. There is no visual problem the reorder icon is displayed as intended at each item in the kirby-reorder-list, moreover I can't find any reference to that icon name in relationsbank repo nor the subsequent folders in the url.
The error appears in staging, when running both locally and otherwise.
I suspect there might be some old path ref in kirby which might cause this?

**Comment by MadsBuchmann** [comment]
@RasmusTraeholt The icons stems from the `ionicons` package as far as i can see. 
I'm honestly a bit in doubt about how we handle icons from that package, i got a couple of questions tho: 

1. Did this happen after an update of Kirby? If so, what is the latest version that worked?
2. Do you have `ionicons` in your `node_modules` folder - and if you do does it contain the missing svg? 
3. If `ionicons` exist which version is installed as identified by: `node_modules/ionicons/package.json`? 

As far as i can see, it does not seem like we currently distribute the svg; perhaps the source of the issue :)

**Comment by RasmusTraeholt** [comment]
It is a very old error, and not related to any resent kirby release. No clue how far back though..
ionicons and the svg does exists in node_modules though. 
I could only find a ref in the package-lock under @ionic/core which states a dependency of ionicons ^5.1.2

**Comment by RasmusTraeholt** [comment]
What drove me to towards kirby, was because it is only triggered when initializing a component that uses the kirby-reorder-list component

**Comment by Joeskov** [comment]
Please notice that the issue appears in production too. But there is no visible error for the end user.
It's seem to appears the first time the reordre list is loaded. When I try to reproduce the issue in cookbook, the "reordre-three-outline.svg" respond with Http:200.

From Cookbook
![image](https://user-images.githubusercontent.com/88030854/160553872-5624ff3a-6531-4288-8996-728811b8ae72.png)

From staging env.
![image](https://user-images.githubusercontent.com/88030854/160554091-ec00b123-65ba-4604-9493-00da163045f0.png)

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
